.custom_wordlist.txt

```diff
@@ -284,6 +284,7 @@ sgsr
 SurfaceFlinger
 swiftshader
 swrast
+synchronization
```

Comment:
Nitpick: this should not be necessary anymore as we're now using en_US by default (see https://github.com/canonical/anbox-cloud-docs/pull/269)